packages/functions/vitest.config.ts

```diff
@@ -5,7 +5,7 @@ export default defineConfig({
     target: 'esnext',
   },
   test: {
-    include: ['src/**/*.test.ts'],
+    include: ['(src|dev)/**/*.test.ts'],
```

Comment:
`packages/functions/dev/main.test.ts` was not running, with this added to config, other than added failing test (expected), it seems like some other tests in this module are failing/timing out :/ https://github.com/netlify/primitives/actions/runs/16414390123/job/46376826143?pr=373#step:11:301